client/dashboard/domains/name-servers/form.tsx

```diff
@@ -203,8 +203,8 @@ export default function NameServersForm( {
 	return (
 		<form onSubmit={ handleSubmit }>
 			<VStack spacing={ 4 }>
-				{ queryError && <Notice variant="error">{ queryError }</Notice> }
-				{ ! queryError && (
+				{ error && <Notice variant="error">{ error }</Notice> }
```

Comment:
Suggestion: since the form does not really show the form when there's error, I'd move this logic out of this form. So it's `client/dashboard/domains/name-servers/index.tsx` who has the responsibility to either display the error notice or the form.

Reply:
True. It makes sense now. I'll remove it from the form.

It used to be different; there was a toggle component, but it doesn't make sense to be there.
<img width="692" height="247" alt="Screenshot 2025-08-15 at 12 07 07" src="https://github.com/user-attachments/assets/5dbdca7b-7846-48f1-9e95-ed1bb873f677" />

Reply:
I moved the Error Notice outside the form. It should be fine now.